### ISSUE TITLE
The problem I faced when the network is trained. I don't know whether it is the problem of torch install.

### ISSUE THREAD

**Issue by NingNingYang** [soumith/dcgan.torch#29]
{
  fineSize : 64
  dataset : "folder"
  batchSize : 64
  nThreads : 4
  noise : "normal"
  niter : 25
  nz : 100
  gpu : 1
  name : "experiment1"
  display_id : 10
  display : 1
  lr : 0.0002
  ngf : 64
  ndf : 64
  beta1 : 0.5
  loadSize : 96
  ntrain : inf
}
Random Seed: 2937	
Starting donkey with id: 4 seed: 2941
table: 0x87f62f0
Starting donkey with id: 3 seed: 2940
table: 0xb3e9f4c0
Starting donkey with id: 1 seed: 2938
table: 0xb3d8a240
Starting donkey with id: 2 seed: 2939
table: 0xb3c42e78
Creating train metadata
table: 0x89ee6d8
Creating train metadata
table: 0xb38abcb8
Creating train metadata
table: 0xb342b968
Creating train metadata
table: 0xb3c9de38
/mnt/hgfs/vmpublic/DCGAN/torch/install/bin/lua: ...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:183: [thread 4 callback] .../hgfs/vmpublic/DCGAN/dcgan.torch-master/data/dataset.lua:139: attempt to index global 'jit' (a nil value)
stack traceback:
	.../hgfs/vmpublic/DCGAN/dcgan.torch-master/data/dataset.lua:139: in function '__init'
	...mpublic/DCGAN/torch/install/share/lua/5.2/torch/init.lua:91: in function <...mpublic/DCGAN/torch/install/share/lua/5.2/torch/init.lua:87>
	[C]: in function 'dataLoader'
	...vmpublic/DCGAN/dcgan.torch-master/data/donkey_folder.lua:82: in main chunk
	[C]: in function 'dofile'
	...mpublic/DCGAN/torch/install/share/lua/5.2/paths/init.lua:84: in function 'dofile'
	/mnt/hgfs/vmpublic/DCGAN/dcgan.torch-master/data/data.lua:42: in function </mnt/hgfs/vmpublic/DCGAN/dcgan.torch-master/data/data.lua:32>
	(...tail calls...)
	[C]: in function 'xpcall'
	...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:234: in function 'callback'
	...blic/DCGAN/torch/install/share/lua/5.2/threads/queue.lua:65: in function <...blic/DCGAN/torch/install/share/lua/5.2/threads/queue.lua:41>
	[C]: in function 'pcall'
	...blic/DCGAN/torch/install/share/lua/5.2/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:183: in function 'dojob'
	...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:264: in function 'synchronize'
	...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:142: in function 'specific'
	...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:125: in function <...ic/DCGAN/torch/install/share/lua/5.2/threads/threads.lua:36>
	(...tail calls...)
	/mnt/hgfs/vmpublic/DCGAN/dcgan.torch-master/data/data.lua:30: in function 'new'
	main.lua:38: in main chunk
	[C]: in function 'dofile'
	...CGAN/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: in ?

**Comment by soumith** [comment]
fixed via https://github.com/soumith/dcgan.torch/commit/f709ad999d8e5f13457b0a0a7c2d7d8c4ffa8696
